pkg/timestream/macros.go

```diff
@@ -10,7 +10,10 @@ import (
 )
 
 const timeFilter = `\$__timeFilter`
+const timeFromStr = `$__time_from`
```

Comment:
I know this file has a mix, but ideally the macros should be camelCase, so it should be `$__timeFrom` and `$__timeTo`